src/framework/uicomponents/qml/Muse/UiComponents/StyledMenuLoader.qml

```diff
@@ -29,11 +29,14 @@ Loader {
     id: loader
 
     signal handleMenuItem(string itemId)
+    signal openPrevMenu()
+    signal openNextMenu()
     signal opened()
     signal closed(bool force)
 
     property alias menu: loader.item
     property var menuAnchorItem: null
+    property bool hasSiblingMenus: loader.hasSiblingMenus
```

Comment:
I think you just have a typo here - you made a property binding to the same property
should be `property bool hasSiblingMenus: ""` and in StyledMenu should be 

`StyledMenu { hasSiblingMenus: loader.hasSiblingMenus }`